### DISCUSSION TITLE
1.1.33

### DISCUSSION THREAD

**Discussion by elihwyma** [wstyres/Zebra#2046] [Releases]
- Fix a crash on iOS 9 -> 11 caused by Sentry mistakingly calling Swift code

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.33'>1.1.33</a>.</em>

**Comment by kdroz19** [comment]
> - Fix a crash on iOS 9 -> 11 caused by Sentry mistakingly calling Swift code
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.33'>1.1.33</a>.</em>

**Reply by elihwyma** [reply]
Cool

**Comment by e-ave** [comment]
What's the point of fixing a bug on iOS 9->11 when the app isn't even installable without iOS 15? I really need this on iOS 10.3.3

**Reply by iAdam1n** [reply]
It is? It can be installed from iOS 9 onwards. Just add the repo, https://getzbra.com/repo.

**Reply by e-ave** [reply]
I cant use cydia. it's broken which is why I need this instead. Cant install openssh or apple file conduit to fix cydia. Was trying to install it with sideloadly and the latest version says ios 15 is required. 1.1.28 worked though but I guess I have to modify the jailbreaking tool itself to install this with the correct permissions
